python-checks/src/main/java/org/sonar/python/checks/UnusedLocalVariableCheck.java

```diff
@@ -53,7 +55,8 @@ public class UnusedLocalVariableCheck extends PythonSubscriptionCheck {
   private static final String DEFAULT = "(_[a-zA-Z0-9_]*|dummy|unused|ignored)";
   private static final String MESSAGE = "Remove the unused local variable \"%s\".";
   private static final String SEQUENCE_UNPACKING_MESSAGE = "Replace unused local variable \"%s\" with \"_\".";
-  private static final String SEQUENCE_UNPACKING_QUICK_FIX_MESSAGE = "Replace with \"_\"";
+  private static final String LOOP_INDEX_MESSAGE = "Replace unused loop index \"%s\" with \"_\".";
```

Comment:
I see you stayed consistent with `SEQUENCE_UNPACKING_MESSAGE `, which is good. However I feel like there is a `the` article missing from both messages.
I'd suggest taking the opportunity to replace both messages with `Replace the unused ...`.